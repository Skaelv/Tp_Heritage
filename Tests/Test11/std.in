PL PL2 12356 127
EXIT
