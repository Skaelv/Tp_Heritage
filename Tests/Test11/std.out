 # Entrez une commande
OK
# New Object : PL2
 # Entrez une commande

